### ISSUE TITLE
Proper dependencies for JRI?

### ISSUE THREAD

**Issue by kenahoo** [s-u/REngine#10]
What's a recommended configuration to use `REngine` version 2.1.0 with JRI?

I currently have the following Maven dependencies in my `pom.xml` file:

        <dependency>
            <groupId>com.github.lucarosellini.rJava</groupId>
            <artifactId>JRIEngine</artifactId>
            <scope>provided</scope>
            <version>0.9-7</version>
        </dependency>
        <dependency>
            <groupId>org.rosuda.REngine</groupId>
            <artifactId>REngine</artifactId>
            <scope>provided</scope>
            <version>2.1.0</version>
        </dependency>

However, the `com.github.lucarosellini.rJava/JRIEngine` artifact depends on `com.github.lucarosellini.rJava/JRI` and `com.github.lucarosellini.rJava/REngine` artifacts, and this latter dependency includes some of the same classes as `org.rosuda.REngine/REngine`, but different versions of them, giving conflicts.

**Comment by s-u** [comment]
Obviously, `com.github.lucarosellini.rJava` has nothing to do with this repo so I can't answer that. However, if packaging JRI for Maven is of interest I'd be happy to entertain PRs, but I don't know how Maven supports JRI binaries as they are system-specific.

**Comment by kenahoo** [comment]
It might be possible to use Travis to generate the build artifacts and make a cross-platform jar.  I'm not much of an expert on that, but I can ask around to people who might know.  Thanks for considering it, if I find anything useful I'll let you know.

**Comment by kenahoo** [comment]
I haven't figured anything out yet, but I just wanted to let you know we're still thinking about how this stuff works.
